definitions/divorce/json/CaseEventToFields/CaseEventToFields-object-to-costs-nonprod.json

```diff
@@ -44,7 +44,7 @@
     "PageID": "judgeCostOrderDecision-1",
     "PageDisplayOrder": 1,
     "PageColumnNumber": 1,
-    "FieldShowCondition": "D8ReasonForDivorceAdulteryWishToName=\"Y*\" OR D8ReasonForDivorceAdulteryIsNamed=\"Y*\""
+    "FieldShowCondition": "D8ReasonForDivorceAdulteryWishToName=\"Y*\" AND JudgeCostsClaimGranted=\"Y*\" OR D8ReasonForDivorceAdulteryIsNamed=\"Y*\" AND JudgeCostsClaimGranted=\"Y*\""
```

Comment:
Seems like parenthesis are not supported so I had to do `A && C || B && C ` instead of `( A || B ) && C `

Reply:
Can't use operator AND and OR together due to CCD limitation hence moving questions to different pages (same as `LA Make a Decision`)